### ISSUE TITLE
iOS Siskin.im app wont connect

### ISSUE THREAD

**Issue by kaisersoze187** [tigase/siskin-im#20]
**Describe the bug**
Siskin.im worked fine until a couple of weeks ago for some reason its stuck and wont connect to my account but the app works just fine on my ipad, if i delete the app and reinstall it on my phone my account is still in the app and if i swipe to delete it a pop says that in order for me to remove the account i have to disable push but i cant because the app wont connect.
![2B4AC3E3-0D92-4D33-80FA-3803A73D1DF9](https://user-images.githubusercontent.com/61890215/76126192-b3e6ec80-5fbb-11ea-9d3d-09bbbc09f631.jpeg)
![3CF60C5F-57FC-42AD-8792-D4203445AAA9](https://user-images.githubusercontent.com/61890215/76126195-b5181980-5fbb-11ea-8b2f-3ee5df88ff49.jpeg)

Iphone 11 pro max 
Ios 13.3.1
Safari

Chatsecure works just fine on my phone what i been using since siskin is down right now please fix this issue

**Comment by hantu85** [comment]
This issue was already reported for version 5.6 and possible fix was released in version 5.7 of Siskin IM. Have you tried this new version?

Please note, that this issue is not easy to replicate and due to that it is not easy to solve as we do not have an easy and fast way to verify if our changes fixed this issue as it occurs randomly.

**Comment by kaisersoze187** [comment]
> This issue was already reported for version 5.6 and possible fix was released in version 5.7 of Siskin IM. Have you tried this new version?
> 
> Please note, that this issue is not easy to replicate and due to that it is not easy to solve as we do not have an easy and fast way to verify if our changes fixed this issue as it occurs randomly.

@hantu85 i can tell you the update did not fix issue

**Comment by woj-tek** [comment]
@kaisersoze187 - could you try to specify more or less when it stopped working? The more exact the date the better.

**Comment by kaisersoze187** [comment]
> @kaisersoze187 - could you try to specify more or less when it stopped working? The more exact the date the better.

Sure i would say maybe 2 weeks before this update had been released, app worked fine then one day it was stuck not connecting

**Comment by woj-tek** [comment]
>  maybe 2 weeks before this update

So around 20th February?

**Comment by kaisersoze187** [comment]
> > maybe 2 weeks before this update
> 
> So around 20th February?

Yes that is correct

**Comment by kaisersoze187** [comment]
This latest update didnt fix the issue, ive decided to uninstall the app and maybe give it another chance in the future

**Comment by hantu85** [comment]
Then it is really hard to tell what was causing the issue in your case. We replicated the issue and fix for it is part of the last release.

**Comment by hantu85** [comment]
How long have you waited on the last version before you decided that it won't connect? It should "reset" within 15s and start a new connection.

**Comment by kaisersoze187** [comment]
I just reinstalled the app and same thing and waiting 3 minutes for it to connect, is there a way you guys can update the app to where the persons account is removed from the app so you have to login again because i think that might be part of the issue i reinstall the app and my account is still in the app

**Comment by woj-tek** [comment]
Thank you for your patience and cooperation. We will include option to remove inactive accounts in future versions.

**Comment by kaisersoze187** [comment]
> Thank you for your patience and cooperation. We will include option to remove inactive accounts in future versions.

The account isnt inactive though i use it everyday, someone needs to be able to remove their account So if my situation happens someone can re-login thats what i believe could fix the problem

**Comment by woj-tek** [comment]
We know and we will add relevant mechanism. The issue stems with de-registration from push - if you don't disable it and remove the account, you could still get push notifications.

But as I said - we will implement a way to handle this use-case.

**Comment by kaisersoze187** [comment]
> We know and we will add relevant mechanism. The issue stems with de-registration from push - if you don't disable it and remove the account, you could still get push notifications.
> 
> But as I said - we will implement a way to handle this use-case.

Sounds awesome cant wait to use the app again

**Comment by kaisersoze187** [comment]
How's the fix coming along?

**Comment by woj-tek** [comment]
It will be implemented in due time, there is currently no ETA.

**Comment by arturhefczyc** [comment]
On the other hand. The connectivity issue seems to be fixed now. We are now beta testing a new version which seems to have a fix for the connection issues. At least for me it works much better.

**Comment by kaisersoze187** [comment]
> On the other hand. The connectivity issue seems to be fixed now. We are now beta testing a new version which seems to have a fix for the connection issues. At least for me it works much better.

Sounds good i hope it fixes the issue, i really enjoyed siskin while it lasted

**Comment by kaisersoze187** [comment]
> On the other hand. The connectivity issue seems to be fixed now. We are now beta testing a new version which seems to have a fix for the connection issues. At least for me it works much better.

Do you guys use testflight for beta testing?

**Comment by arturhefczyc** [comment]
Yes, we do. If you want to be added to testflight, please use a form on our main website to send us a message with your icloud id.

**Comment by kaisersoze187** [comment]
> Yes, we do. If you want to be added to testflight, please use a form on our main website to send us a message with your icloud id.

Could you put a link up for this "form" because i cant find it on siskin im site or tigase

**Comment by arturhefczyc** [comment]
Follow this link: https://tigase.net/technical-support
On the third position a form opens up. Fill it up and send over. We will take care of the rest.

**Comment by kaisersoze187** [comment]
> Follow this link: https://tigase.net/technical-support
> On the third position a form opens up. Fill it up and send over. We will take care of the rest.

Thank you i sent the form

**Comment by arturhefczyc** [comment]
Added you to beta testers. You should receive email from appstore.

**Comment by kaisersoze187** [comment]
> Added you to beta testers. You should receive email from appstore.

Thank you i installed it but still the same issue, i believe im done till the fix is implemented thank you all for trying to help me

**Comment by kaisersoze187** [comment]
What needs to be done is the app needs to clear it self when its deleted, when you reinstall the app it already will have your account added that needs to be changed where its like a fresh install each time the app is reinstalled
